docs/scripts_md/File.md

```diff
@@ -0,0 +1,178 @@
+# NAME
+
+NeuroDB::File -- Provides an interface to the MRI file management subsystem of
+LORIS
+
+# SYNOPSIS
+
+    use NeuroDB::File;
+    use NeuroDB::DBI;
+
+    my $dbh = NeuroDB::DBI::connect_to_db();
+
+    my $file = NeuroDB::File->new(\$dbh);
+
+    my $fileID = $file->findFile('/path/to/some/file');
+    $file->loadFile($fileID);
+
+    my $acquisition_date = $file->getParameter('acquisition_date');
+    my $parameters_hashref = $file->getParameters();
+
+    my $coordinate_space = $file->getFileDatum('CoordinateSpace');
+    my $filedata_hashref = $file->getFileData();
+
+
+    # less common to use methods, available mainly for register_db...
+    my $dbh_copy = $file->getDatabaseHandleRef();
+
+    $file->loadFileFromDisk('/path/to/some/file');
+    $file->setFileData('CoordinateSpace', 'nonlinear');
+    $file->setParameter('patient_name', 'Larry Wall');
+
+    my $parameterTypeID = $file->getParameterTypeID('patient_name');
+
+# DESCRIPTION
+
+This class defines a BIC MRI (or related) file (minc, bicobj, xfm,
+etc) as represented within the LORIS database system.
+
+**Note:** if a developer does something naughty (such as leaving out
+the database handle ref when instantiating a new object or so on) the
+class will croak.
+
+## Methods
+
+### new(\\$dbh) >> (constructor)
```

Comment:
do you like the ' >> (constructor)' in the h3 title?
Maybe in the line below (in the text), say it is a constructor?

Reply:
It was how this was originally documented. I can remove the '>> (constructor)' part. These classes were already pretty well documented so I kept most of what was already there. Let me know if you want me to remove it. I have no preference.